### PR TITLE
DEV: Fix flaky instagram onebox spec by not mutating constant.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: lib/onebox/engine.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def options=(opt)
 
     def initialize(url, timeout = nil)
       @errors = {}
-      @options = DEFAULT
+      @options = DEFAULT.dup
```
Comment (tgxworld) [inline lib/onebox/engine.rb:45]: Mutation was happening in https://github.com/discourse/discourse/blob/c161f5e0b298fa6842b83d2bc4627b5a8f92aff0/lib/onebox/engine.rb#L39

Comment (martin-brennan) [inline lib/onebox/engine.rb:45]: What is the point of even having this in a const? Can't we just do `@options = {}`?

Comment (tgxworld) [reply lib/onebox/engine.rb:45]: Good point, let me remove it to see if tests fail. We don't need to keep backwards compatibility with this constant anymore.

**Comment by tgxworld** [comment]
@martin-brennan Done :)

File: lib/onebox/engine.rb (hunk 1 above)
Comment (martin-brennan) [reply lib/onebox/engine.rb]: Cool thanks. Just seems really weird to me to even make this into a const in the first place!

**Review by martin-brennan — APPROVED** [review]

**Pull request merged**